shell/platform/glfw/flutter_glfw.cc

```diff
@@ -618,6 +628,22 @@ static bool RunFlutterEngine(
   args.command_line_argv = &argv[0];
   args.platform_message_callback = EngineOnFlutterPlatformMessage;
   args.custom_task_runners = &task_runners;
+
+  if (FlutterEngineRunsAOTCompiledDartCode()) {
```

Comment:
There's already a lot going on in this function, and this is non-trivial code, so I'd strongly prefer it be extracted to a method (see https://github.com/flutter/engine/pull/18589 where I stole this, but made it a method :) )